src/bin/print_even.c

```diff
@@ -0,0 +1,19 @@
+int main();
+int _start() {
+    return main();
+}
+
+#include "syscall.h"
+
+int main() {
+    int i = 0;
+    for (i = 0; i < 11; i+=2) {
+        debug_print(i);
+        int j;
+        for (j = 0; j < 500000000; j++) {} // busywait
+    }
+
+    exit(0);
```

Comment:
We return because main demands a return value, we exit because I don't know how to get exit code to run automatically after a process exits yet.